docs/en/sql-reference/functions/string-functions.md

```diff
@@ -2532,3 +2532,37 @@ Result:
 │ foo                        │
 └────────────────────────────┘
 ```
+## strncmp
```

Comment:
Only a minority of ClickHouse users are C developers. Can you please give this function a speaking and human-readable name. Feel free to provide an additional function alias "strncmp".

EDIT: I am looking at MySQL, they have a function [STRCMP](https://dev.mysql.com/doc/refman/8.4/en/string-comparison-functions.html#function_strcmp). Note that it does not have offsets or max_length parameters.

I think we should call the function `compare` and it should have the 1st and 2nd strings as mandatory arguments, followed by the 1st and 2nd offsets and the number of bytes to compare as optional parameters.